src/Illuminate/Cache/DatabaseStore.php

```diff
@@ -100,9 +100,9 @@ public function put($key, $value, $minutes)
         $expiration = $this->getTime() + (int) ($minutes * 60);
 
         try {
-            $this->table()->insert(compact('key', 'value', 'expiration'));
+            return $this->table()->insert(compact('key', 'value', 'expiration'));
         } catch (Exception $e) {
-            $this->table()->where('key', $key)->update(compact('value', 'expiration'));
+            return $this->table()->where('key', $key)->update(compact('value', 'expiration'));
```

Comment:
This returns an integer representing the number of rows affected by the query so this has to be casted to bool.